packages/babel-plugin-debug-ids/src/index.test.ts

```diff
@@ -292,15 +298,92 @@ describe('babel plugin', () => {
     `;
 
     expect(transform(source)).toMatchInlineSnapshot(`
-      "import { globalKeyframes } from '@vanilla-extract/css';
+      import { globalKeyframes } from '@vanilla-extract/css';
       globalKeyframes('myKeyframes', {
         from: {
           transform: 'rotate(0deg)'
         },
         to: {
           transform: 'rotate(360deg)'
         }
-      });"
+      });
+    `);
+  });
+
+  it('should handle layer assigned to const', () => {
+    const source = `
+      import { layer } from '@vanilla-extract/css';
+
+      const autoDebugId = layer();
+      const providedDebugId = layer('utils');
+    `;
+
+    expect(transform(source)).toMatchInlineSnapshot(`
+      import { layer } from '@vanilla-extract/css';
+      const autoDebugId = layer("autoDebugId");
+      const providedDebugId = layer('utils');
+    `);
+  });
+
+  it('should handle layer assigned to a variable', () => {
+    const source = `
+      import { layer } from '@vanilla-extract/css';
+
+      let reset;
+      reset = layer();
+    `;
+
+    expect(transform(source)).toMatchInlineSnapshot(`
+      import { layer } from '@vanilla-extract/css';
+      let reset;
+      reset = layer("reset");
+    `);
+  });
+
+  it('should handle layer with a parent', () => {
+    const source = `
+      import { layer } from '@vanilla-extract/css';
+
+      const reset = layer({ parent: 'papa' });
+      const providedDebugId = layer({ parent: 'papa' }, 'utils');
+    `;
+
+    expect(transform(source)).toMatchInlineSnapshot(`
+      import { layer } from '@vanilla-extract/css';
+      const reset = layer({
+        parent: 'papa'
+      }, "reset");
+      const providedDebugId = layer({
+        parent: 'papa'
+      }, 'utils');
+    `);
+  });
+
+  it('should handle globalLayer', () => {
+    const source = `
+      import { globalLayer } from '@vanilla-extract/css';
+
+      globalLayer('reset');
+    `;
+
+    expect(transform(source)).toMatchInlineSnapshot(`
+      import { globalLayer } from '@vanilla-extract/css';
+      globalLayer('reset');
+    `);
+  });
+
+  it('should handle globalLayer with a parent', () => {
+    const source = `
+      import { globalLayer } from '@vanilla-extract/css';
+
+      const reset = globalLayer({ parent: 'papa' });
+    `;
+
+    expect(transform(source)).toMatchInlineSnapshot(`
+      import { globalLayer } from '@vanilla-extract/css';
+      const reset = globalLayer({
+        parent: 'papa'
+      });
```

Comment:
Should these calls to `globalLayer` have a `name` argument? The type implies that they're mandatory https://github.com/vanilla-extract-css/vanilla-extract/blob/a3a420cf07c6f487564532d04a4410a8b658731f/packages/css/src/layer.ts#L40